tools/code_format/yaml_lint.py

```diff
@@ -0,0 +1,21 @@
+import subprocess
+import sys
+
+YAMLLINT_CONFIG = ".yamllint"
+
+# explicitly use python3 linter
+YAMLLINT_COMMAND = ("python3", "-m", "yamllint", "-c", YAMLLINT_CONFIG, ".")
```

Comment:
not sure the yamllint config setting is necessary here - its changing to the directory - ie envoy roo